tests/dags/common/test_sql_helpers.py

```diff
@@ -0,0 +1,128 @@
+import logging
+from datetime import datetime, timedelta
+
+import pytest
+from common.constants import DAG_DEFAULT_ARGS, POSTGRES_CONN_ID
+from common.sql_helpers import PGExecuteQueryOperator, PostgresHook
+from psycopg2.errors import QueryCanceled
+
+from tests.dags.common.test_resources.dags.test_timeout_pg import (
+    TEST_SQL,
+    create_pg_timeout_tester_dag,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def happy_dag():
+    return create_pg_timeout_tester_dag()
+
+
+DEFAULT_TIMEOUT = timedelta(hours=1).total_seconds()
+default_timeout_msg = (
+    f"DAG_DEFAULT_ARGS sets the default task execution timeout to "
+    f"{DAG_DEFAULT_ARGS['execution_timeout']}. Testing for {DEFAULT_TIMEOUT} seconds."
+)
+
+
+# PostgresHook works with or without explicit connection id, and with override conn id
+# Testing mainly for documentation purposes
+# Not using pg.run("select 1;") to test the actual connection, because we don't spin up
+# the databases for testing.
```

Comment:
We do have a database for testing though, and this could be called! We do other operations with the database as part of our tests.